app/team.go

```diff
@@ -572,14 +572,14 @@ func (a *App) RemoveUserFromTeam(teamId string, userId string) *model.AppError {
 		user = result.Data.(*model.User)
 	}
 
-	if err := a.LeaveTeam(team, user); err != nil {
+	if err := a.LeaveTeam(team, user, requestorId); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (a *App) LeaveTeam(team *model.Team, user *model.User) *model.AppError {
+func (a *App) LeaveTeam(team *model.Team, user *model.User, requestorId string) *model.AppError {
```

Comment:
Would it make sense to instead pass a boolean for whether or not the user was removed from the channel? That's just a thought though. I don't have any strong feelings either way.

Reply:
That would imply to move the comparison between requestorId and user.Id to the caller function in order to get that boolean. I does not really make a big difference in my opinion.